src/op_analytics/transforms/interop/parking/07_export_token_create_traces_v1.sql

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
I'm wondering instead of creating a small subset of just interop tokens created, maybe we can get the 

erc20 created with joins to erc20 first transfers and then left join to the three dim tables with a new column name `token_type`? 

Tho I think the only problem would be that it might omit some of erc20 tokens created before 2024-10-1 (if it's never been transferred again) given we are only starting to compute the first transfers from that date (but I think that should be fine for now and going forward given the purpose of this exercise here).

And any token created before 2024-10-1 that are still being transferred will be recorded still by joining it back to `create_trace`.